_pages/amitbasuri.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: amitbasuri
+pageUrl: amitbasuri.github.io/amitbasuri
+timestamp: 2017-03-23
+---
```

Comment:
I believe it's because you added `\n` instead of an actual new line character. Try editing this right from GitHub.com, and just press enter after line 5.